must-gather/collection-scripts/pre-install.sh

```diff
@@ -65,8 +72,10 @@ check_for_helper_pod(){
 deploy
 labels
 pids=()
-check_for_helper_pod &
-pids+=($!)
+if [ -n "${storageClusterPresent}" ]; then
```

Comment:
-z

Reply:
https://stackoverflow.com/questions/18096670/what-does-z-mean-in-bash